dts/bindings/mipi-dbi/zephyr,mipi-dbi-spi.yaml

```diff
@@ -27,6 +27,22 @@ properties:
     description: |
       Reset GPIO pin. Set high to reset the display
 
+  xfr-min-bits:
```

Comment:
One other (nonblocking) thought- we can use a string enum here, and use `DT_STRING_TOKEN` within the driver to access the property, similar to what was done here: #81293

Reply:
No worries, I am fine with the PR as is- we can always make this change in the future if desired, but it isn't a high priority to clean up (IMO)